internal/infrastructure/kubernetes/proxy/testdata/daemonsets/disable-prometheus.yaml

```diff
@@ -78,19 +78,11 @@ spec:
                 - name: envoy.filters.network.http_connection_manager
```

Comment:
should this listener exist if prometheus is disabled (test case) and ready listener is moved to dynamic listeners?